lib/app.tsx

```diff
@@ -85,6 +86,7 @@ function mapDispatchToProps(dispatch, { noteBucket }) {
       dispatch(actionCreators.setSearchFocus({ searchFocus: true })),
     setSimperiumConnectionStatus: connected =>
       dispatch(toggleSimperiumConnectionStatus(connected)),
+    setSelectedNote: id => dispatch(setSelectedNote(id)),
```

Comment:
The use looks correct but this signature looks misleading…did we mean to put `setSelectedNote: note => dispatch(setSelectedNote(note))`?

Reply:
ooops

Reply:
removed in: d14a6e9